spec/coffee/directives/api/utils/_async.spec.coffee

```diff
@@ -2,21 +2,23 @@ describe "_async", ->
   rootScope = null
   timeout = null
   q = null
+  browser = null
 
   digest = (fn, times = 1) =>
     fn()
     if times
       _.range(times).forEach -> # i would like to say that it sucks that I have to do this.. (angular)
-        timeout?.flush()
+        timeout?.flush() if browser.deferredFns?.length
```

Comment:
what about:
```
fn()
rootScope?.$digest()
while !timeout?.verifyNoPendingTasks()
  timeout?.flush()
  rootScope?.$digest()
```
I think that should eliminate the need to pass in `times`.  I'm not sure if it's necessary to put the digest call inside the loop like that, but if the function that's being called via timeout is setting up another timeout, then it wouldn't surprise me if it is.